### PR TITLE
Mauve

### PR DESCRIPTION
**Pull request by jamesabbott**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

New recipe for the final component of Mauve

### CODE REVIEW

**Comment by bgruening** [comment]
Oh it is green!!!

**Comment by bgruening** [comment]
Ready to merge?

**Comment by jamesabbott** [comment]
Not quite...The test is overly basic and I don't actually now it works. It's a java gui app which has some command-line only methods but even with -Djava.awt.headless=true functional tests fail since it looks for X11. A bit more playing to do yet....If I can't get a headless test to work I guess it will have to be merged and then tested manually when the build is available

**Comment by jamesabbott** [comment]
My Attempte to improve the tests have not succeeded. Testing with openjdk fails with errors accessing fonts even when using conda-forge::openjdk and freetype as discussed in issue #8586. Installing openjdk from conda-forge and testing a build against this works ok, so hopefully it is working...

Think it will have to be merged as-is, unless anyone has some ideas as to how to fix the java font issue...

**Comment by corburn** [comment]
If xvfb is available, `xvfb-run [command]` could be used to run the program in a virtual xserver.

File: recipes/mauve/meta.yaml [hunk 1]
```diff
@@ -16,11 +16,11 @@ source:
 requirements:
   build:
     - ant
-    - conda-forge::openjdk
+    - "conda-forge::openjdk"
   host:
     - mauvealigner
   run:
-    - conda-forge::openjdk
+    - "conda-forge::openjdk"
```
Comment (bgruening) [inline recipes/mauve/meta.yaml]: If its still installing from `defaults` which is already very wired, I think another dependency has a pinning to the default one or something like that.
By default openjdk should come from conda-forge. Or maybe the conda-forge one is broken and we should fix this one.

Comment (jamesabbott) [reply recipes/mauve/meta.yaml]: I've tried quoting 'conda-forge::openjdk' as well but it still comes from defaults. Pinning the version number as fastqc does (issue #8586) solves the problem, but strikes me as being a little fragile.

From the previous issues, the conda-forge jdk is the one that works, and the one in defaults is broken but is tending to get pulled in.

Anyway...I _finally_ have a functioning test, so it's ready to merge if it looks ok. Try not to laugh at the number of commits it took to get to this point...

**Comment by bgruening** [comment]
Thanks @jamesabbott for all your effort!

**Pull request merged**